### ISSUE TITLE
I pledge to live my the Contributor Covenant

### ISSUE THREAD

**Issue by samboy** [samboy/MaraDNS#46]
I hereby pledge, as a move forward with MaraDNS contributions, to live by the [Contributor Covenant](https://www.contributor-covenant.org/version/1/4/code-of-conduct)

To wit, I will be empathetic. I will avoid saying hurtful things, but if I do, I strive to apologize, even I feel I did nothing wrong. I will be respectful. I will avoid objectifying women, as per my personal religious convictions as as per said covenant. I will avoid saying phrases from the 1980s, 1990s, and first 2000s decades (and even in the early 2010s) like RTFM (read the manual) or STFW (search the web). 

In other words, the days of being an old *NIX grouse should be a thing of the past. Now, I get to be an old jolly open source developer. Well, may not that old yet, but, yes, my beard is starting to become gray.

**Comment by samboy** [comment]
I have added CODE_OF_CONDUCT.md to all of my GitHub projects which are not simple mirrors of other projects. This is how I will make this pledge part of my future open source conduct. I am getting too old to be a grousy RTFM-screaming *NIX greybeard.

**Comment by samboy** [comment]
Locking conversation: I do not think this is a topic which leads to productive debate.

**Comment by samboy** [comment]
Let me clarify here: This does not mean I will take political correctness to the point of being unkind (example: The far-left extremists who wish to tarnish Richard Feynman’s legacy for things he wrote about that they interpret in the most unkind way possible).  This means, very simply: “Be Kind”.  Do not say hateful things about a group or class of people. Do not say hateful or unkind things. Do not insult people’s intelligence (even if they don’t know something which seems obvious to me). Do not cast shade on or be negative. Be kind. Be positive. Be loving.
